Doc/library/csv.rst

```diff
@@ -166,8 +166,8 @@ The :mod:`csv` module defines the following classes:
    All other optional or keyword arguments are passed to the underlying
    :class:`reader` instance.
 
-   .. versionchanged:: 3.6
-      Returned rows are now of type :class:`OrderedDict`.
+   .. versionchanged:: 3.8
+      Returned rows are now of type :class:`dict`.
```

Comment:
This entry should have been added without deleting the previous one

Reply:
Looks like this never got fixed...

Reply:
PR #20657 fixes this